config/gcos-strict.conf

```diff
@@ -39,6 +41,9 @@ pretty-display:			yes
 # Allow complex OCCURS DEPENDING ON
 complex-odo:			no
 
+# Adjust position of items following OCCURS DEPENDING
```

Comment:
I think these should belong to another pull request ;-)

Reply:
To be honest, it's very likely that will be squashing all these PRs into a single commit ("add GCOS dialect") sent as a single patch to the official GNUCobol team. So, no need to be extract careful about splitting PRs and commits for now. We may split it at some point to extract things that are not specifically tied to GCOS.